src/crimson/os/seastore/cache.cc

```diff
@@ -21,8 +21,9 @@ namespace {
 
 namespace crimson::os::seastore {
 
-Cache::Cache(SegmentManager &segment_manager) :
-  segment_manager(segment_manager) {}
+Cache::Cache(SegmentManager &segment_manager, PerfCounters *perf) :
+  segment_manager(segment_manager),
+  c_perf(perf) {}
```

Comment:
c_perf appears unused?

Reply:
have we looked into seastar's metric subsystem? is it a better alternative in comparison to `PerfCounters`?

Reply:
> c_perf appears unused?

not yet.  currently no statistic information in your list need be collected from cache.  In case we need it later.  I can remove it in this PR if you think so.

Reply:
We can add it once there's a user.  Remove it for now.

Reply:
> have we looked into seastar's metric subsystem? is it a better alternative in comparison to `PerfCounters`?

yingxin seems try it. I have no idea about it now.